src/App.jsx

```diff
@@ -1,22 +1,38 @@
-import './App.css';
-import React, { useState } from 'react';
-import TodoList from './TodoList';
-import AddTodoForm from './AddTodoForm';
+import React, { useState, useEffect } from 'react';
+
+function useSemiPersistentState(key, initialValue) {
+  const [value, setValue] = useState(() => {
+    const storedValue = localStorage.getItem(key);
+    return storedValue ? JSON.parse(storedValue) : initialValue;
+  });
+
+  useEffect(() => {
+    localStorage.setItem(key, JSON.stringify(value));
+  }, [value, key]);
+
+  return [value, setValue];
+}
 
 function App() {
-    const [todoList, setTodoList] = useState([]);
-
-    const addTodo = (newTodo) => {
-        setTodoList((prevTodoList) => [...prevTodoList, newTodo]);
-    };
-
-    return (
-        <div>
-            <h1>My Todo List</h1>
-            <AddTodoForm onAddTodo={addTodo} />
-            <TodoList todoList={todoList} />
-        </div>
-    );
+  const [todoList, setTodoList] = useSemiPersistentState('todoList', []);
+
+  const addTodo = (newTodo) => {
+    setTodoList([...todoList, newTodo]);
+  };
+
+  const removeTodo = (index) => {
```

Comment:
For removing the item we should use the filter method. In next lesson this will be explained. For now I think it's better to remove this code. It doesn't work.